### PR TITLE
Update Douban.com.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

File: src/chrome/content/rules/Douban.com.xml [hunk 1]
```diff
@@ -1,14 +1,45 @@
...
+    <target host="read.douban.com" />
+    <target host="shanghai.douban.com" />
+    <target host="site.douban.com" />
+    <target host="wap.douban.com" />
+    <target host="www.douban.com" />
+    <target host="ypy.douban.com" />
+    
+	<!--
+    movie.douban.com" />
+    (Automatic redirection to HTTP)
```
Comment (J0WI) [inline src/chrome/content/rules/Douban.com.xml]: WFM

File: src/chrome/content/rules/Douban.com.xml [hunk 2]
```diff
@@ -1,14 +1,44 @@
...
+    -->
+
+    <!--
+    pypi.douban.com" />
+    (Mismatch)
```
Comment (nextPrime) [inline src/chrome/content/rules/Douban.com.xml]: No longer a mismatch: https://pypi.douban.com/

**Comment by J0WI** [comment]
Thanks for all the corrections. Could you please use tabs instead of spaces to match our code style?
I'll merge it then.

**Comment by J0WI** [comment]
let me rerun the tests

**Comment by J0WI** [comment]
We have a very slow connection to this server, but it finally passed all tests.

**Pull request merged**